include/hip/hip_runtime_api.h

```diff
@@ -116,7 +116,10 @@ typedef struct hipDeviceProp_t {
     int gcnArch;                              ///< AMD GCN Arch Value. Eg: 803, 701
     int integrated;            ///< APU vs dGPU
     int cooperativeLaunch;            ///< HIP device supports cooperative launch
-    int cooperativeMultiDeviceLaunch; ///< HIP device supports cooperative launch on multiple devices 
+    int cooperativeMultiDeviceLaunch; ///< HIP device supports cooperative launch on multiple devices
```

Comment:
This is not backward compatible.

A HIP app can define a variable of hipDeviceProp_t. A HIP app built before this change will have a hipDeviceProp_t variable without these fields and will corrupt memory when runs with HIP runtime built after this change. Actually we saw PyTorch tests failing when running PyTorch built before this change with HIP runtime built after this change.